apps/registry/codemods/react/19/remove-memoization-hooks/src/index.ts

```diff
@@ -0,0 +1,46 @@
+import type { API, FileInfo } from "jscodeshift";
+
+export default function transform(
+	file: FileInfo,
+	api: API,
+): string | undefined {
+	const j = api.jscodeshift;
+	const root = j(file.source);
+
+	const hooksToRemove = ["useMemo", "useCallback", "memo"];
+
+	root.find(j.ImportDeclaration).forEach((path) => {
+		const specifiers =
+			path.node.specifiers?.filter((specifier) => {
+				if (specifier.type === "ImportSpecifier") {
+					return !hooksToRemove.includes(specifier.imported.name);
+				}
+				return specifier;
+			}) ?? [];
+
+		if (specifiers.length === 0) {
+			j(path).remove();
+		} else {
+			path.node.specifiers = specifiers;
+		}
+	});
```

Comment:
you are removing each import that has no import specifiers. this would remove side-effect imports, such as `import "file.css"`
i would try removing this part, as you are removing identifiers below. you probably don't even need to clear the import statements.

Reply:
> you are removing each import that has no import specifiers. this would remove side-effect imports, such as import "file.css"

Good point! I pushed a fix.